rg_shipit/shipit_service/ShipitCityTrack.php

```diff
@@ -0,0 +1,47 @@
+<?php
+  class ShipitCityTrack {
+    public $draft = '';
+    public $confirmed = '2019-06-07T17:13:09.141-04:00';
```

Comment:
Just curious. What is this date about?

Reply:
in the original model it was like this, I kept it, but it was one of the two classes that is no longer used but remains